### PR TITLE
{bio}[foss/2022a] CellChat v1.5.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node302.hydra.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/smoors/6dc4ac09dffb8730b9c169829f7899da for a full test report.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/7c8a97b0bab5507621cec8710b7dbada for a full test report.

File: easybuild/easyconfigs/c/CellChat/CellChat-1.5.0-foss-2022a-R-4.2.1.eb [hunk 1]
```diff
@@ -0,0 +1,57 @@
+easyblock = 'Bundle'
+
+name = 'CellChat'
+version = '1.5.0'
+versionsuffix = '-R-%(rver)s'
```
[the remark this answers is not in the document]
Comment (smoors) [reply easybuild/easyconfigs/c/CellChat/CellChat-1.5.0-foss-2022a-R-4.2.1.eb]: as discussed here https://github.com/easybuilders/easybuild-easyconfigs/pull/17722#pullrequestreview-1385882441 we have easyconfigs with and without the versionsuffix, and it's not enforced.

but indeed makes sense to remove it, done.

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/1a38bcef6243531da562599eed460696 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
Going in, thanks @smoors!

**Pull request merged**